utils/dwarf.c

```diff
@@ -1288,7 +1407,12 @@ static void save_debug_entries(struct debug_info *dinfo,
 			continue;
 
 		save_debug_file(fp, 'F', loc->sym->name, loc->sym->addr);
-		save_debug_file(fp, 'L', loc->file->name, loc->line);
+
+		if (dinfo->base_dir) {
+			if (strstr(loc->file->name, dinfo->base_dir))
+				idx = strlen(dinfo->base_dir) + 1;
+		}
+		save_debug_file(fp, 'L', loc->file->name + idx, loc->line);
```

Comment:
Please add a comment that it skips the common part.

Reply:
ok. thank you.